spec/views/admin/index.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'admin/index' do
+  let(:ability) { Ability.new(user) }
+  before do
+    allow(controller).to receive(:current_ability).and_return(ability)
+    render
+  end
+  context "when logged in as an admin" do
+    let(:user) { FactoryGirl.build(:admin) }
```

Comment:
There's no magic at work. I didn't comment about it because I figured it'd be good for the future - there's a good chance what links you see will dependent on what role you have.